numpy/core/defchararray.py

```diff
@@ -1908,7 +1908,7 @@ class adds the following functionality:
     unicode : bool, optional
         Are the array elements of type unicode (True) or string (False).
         Default is False.
-    buffer : int, optional
+    buffer : int or str or bytes, optional
```

Comment:
Is int actually allowed here?

Reply:
Without wanting to check, I somewhat expect this was that ominous `int_asbuffer` function that we removed in the p3k cleanup :).

Reply:
no, it is not allowed. `np.chararray((3, 3), buffer=12)` emits `TypeError: a bytes-like object is required, not 'int'`